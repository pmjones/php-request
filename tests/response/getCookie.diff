001+ Fatal error: Uncaught Error: Call to undefined method SapiResponse::getCookie() in /Users/pmjones/Code/pmjones/php-request/tests/response/getCookie.php:4
002+ Stack trace:
003+ #0 {main}
004+   thrown in /Users/pmjones/Code/pmjones/php-request/tests/response/getCookie.php on line 4
001- array(8) {
002-   ["value"]=>
003-   string(6) "v1&%v2"
004-   ["expires"]=>
005-   int(0)
006-   ["path"]=>
007-   string(0) ""
008-   ["domain"]=>
009-   string(0) ""
010-   ["secure"]=>
011-   bool(false)
012-   ["httponly"]=>
013-   bool(false)
014-   ["samesite"]=>
015-   string(0) ""
016-   ["url_encode"]=>
017-   bool(true)
018- }
019- NULL