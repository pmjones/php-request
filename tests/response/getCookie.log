
---- EXPECTED OUTPUT
array(8) {
  ["value"]=>
  string(6) "v1&%v2"
  ["expires"]=>
  int(0)
  ["path"]=>
  string(0) ""
  ["domain"]=>
  string(0) ""
  ["secure"]=>
  bool(false)
  ["httponly"]=>
  bool(false)
  ["samesite"]=>
  string(0) ""
  ["url_encode"]=>
  bool(true)
}
NULL
---- ACTUAL OUTPUT
Fatal error: Uncaught Error: Call to undefined method SapiResponse::getCookie() in /Users/pmjones/Code/pmjones/php-request/tests/response/getCookie.php:4
Stack trace:
#0 {main}
  thrown in /Users/pmjones/Code/pmjones/php-request/tests/response/getCookie.php on line 4
---- FAILED
