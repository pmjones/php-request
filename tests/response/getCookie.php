<?php
$response = new SapiResponse();
$response->setCookie('cookie1', 'v1&%v2');
var_dump($response->getCookie('cookie1'));
var_dump($response->getCookie('cookie2'));
