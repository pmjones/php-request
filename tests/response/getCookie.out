Fatal error: Uncaught Error: Call to undefined method SapiResponse::getCookie() in /Users/pmjones/Code/pmjones/php-request/tests/response/getCookie.php:4
Stack trace:
#0 {main}
  thrown in /Users/pmjones/Code/pmjones/php-request/tests/response/getCookie.php on line 4